backend/models/events/events-model.js

```diff
@@ -1,14 +1,15 @@
 const mongoose = require("mongoose")
 const { Schema } = mongoose
-const imageModel = require("../image/image-model")
+const image = require("../image/image-model")
+const imageSchema = mongoose.model("image").schema
 
 // Add your code here
 const eventSchema = new Schema({
     title: {
         type: String,
         required: true,
     },
-    description: {
+    detail: {
```

Comment:
The fakeData has this field listed as "description" right now. Recommend to keep it all consistent to "description" right now, unless you want to change it all to "detail".

Reply:
I changed `description` into `detail`